cmd/oras/root/manifest/index/create.go

```diff
@@ -56,7 +56,7 @@ type createOptions struct {
 func createCmd() *cobra.Command {
 	var opts createOptions
 	cmd := &cobra.Command{
-		Use:   "create [flags] <name>[:<tag[,<tag>][...]] [{<tag>|<digest>}...]",
+		Use:   "create [flags] <target>[:<tag[,<tag>][...]] <sources>[{<tag>|<digest>}...]",
```

Comment:
Just realized that maybe we should still call it `<name`> to be consistent with other commands. What do you think? @wangxiaoxuan273

Reply:
I think `<target>` is better since it pairs with `sources`.